doc/quick_start_guide.md

```diff
@@ -72,8 +74,8 @@ cygnusagent.sources.http-source.handler.default_service_path = def_servpath
 cygnusagent.sources.http-source.handler.events_ttl = 2
 cygnusagent.sources.http-source.interceptors = ts de
 cygnusagent.sources.http-source.interceptors.ts.type = timestamp
-cygnusagent.sources.http-source.interceptors.de.type = com.telefonica.iot.cygnus.interceptors.DestinationExtractor$Builder
-cygnusagent.sources.http-source.interceptors.de.matching_table = /Applications/apache-flume-1.4.0-bin/conf/matching_table.conf
+cygnusagent.sources.http-source.interceptors.de.type = com.telefonica.iot.cygnus.interceptors.GroupingInterceptor$Builder
+cygnusagent.sources.http-source.interceptors.de.matching_table = /Applications/apache-flume-1.4.0-bin/conf/grouping_rules.conf
```

Comment:
Yes, but there is no easy way to fix it: the `com.telefonica.iot.cygnus.interceptors.GroupingInterceptor$Builder` is the name of a class, and it is automatically taken by Apache Flume to instantiate it... there is no code controlled by us allowing to say "if appearing the old class name, instantiate the new one instead".

BTW, I've noticed there is another change in the configuration, I'll upload another commit.